pkg/build/daggerbuild/artifacts/package_targz.go

```diff
@@ -55,9 +55,7 @@ type Tarball struct {
 	// Dependent artifacts
 	Backend        *pipeline.Artifact
 	Frontend       *pipeline.Artifact
-	NPMPackages    *pipeline.Artifact
```

Comment:
Doesn't the full core build depend on the built npm artifacts?

Reply:
they're requested separately not as part of the targz